### PR TITLE
feat: Adding icmp to Instcombine's PrettyEDSL

### PR DESCRIPTION
**Pull request by lfrenot**

### CODE REVIEW

**Comment by tobiasgrosser** [comment]
Is this now final, @lfrenot?

File: SSA/Projects/InstCombine/scripts/cfg.py [hunk 1]
```diff
@@ -17,6 +17,11 @@
 proof_path = "SSA/Projects/InstCombine/tests/proofs"
 log_path = "SSA/Projects/InstCombine/tests/logs"
 llvm_test_path = llvm_path + "/llvm/test/Transforms/InstCombine"
+
+test_path = "SSA/Projects/InstCombine/scripts/files/LLVM"
+proof_path = "SSA/Projects/InstCombine/scripts/files/proofs"
+log_path = "SSA/Projects/InstCombine/scripts/files/logs"
+llvm_test_path = "SSA/Projects/InstCombine/scripts/files"
```
Comment (tobiasgrosser) [inline SSA/Projects/InstCombine/scripts/cfg.py]:
```suggestion
```

This is unrelated.

File: SSA/Projects/InstCombine/scripts/cfg.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 from enum import Enum, auto
 
 # The path from lean-mlir to llvm-project
-llvm_path = "../llvm-project-main"
+llvm_path = "../llvm-project"
```
Comment (tobiasgrosser) [inline SSA/Projects/InstCombine/scripts/cfg.py]:
```suggestion
llvm_path = "../llvm-project-main"
```

This change seems unrelated.

**Comment by lfrenot** [comment]
Yes, this should be final

**Comment by lfrenot** [comment]
We're good to merge
